CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```

Comment:
Hi @GeetDsa .
Thank you for contributing to Pycm :+1:.
@sepandhaghighi meant to add `- Weighted average for various class stats` line here (under `Unreleased` tag) because your changes will be published for next release.
We'll transfer changes from `Unreleased` tag to next release.